### PR TITLE
Updated README with more explicit instructions on how to set up node.…

### PR DESCRIPTION
**Pull request by skantzidis**
Updated README with information on how to install change-case module dependency, as this was previously missed in last README updates. Also included link to node.js, as it was obvious when working with another team member that it was not obvious that installation of node.js was required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,25 @@ automate parts that require a bit more programmatic control.
...
 ## Prerequisites
 
+### Install node.js
+
+Install node.js. See [here](https://nodejs.org/en/).
+
 ### Node and Handlebars
 
-This has been tested with Node v8.1.2 and Handlebars v4.0.10.
+This has been tested with Node v8.1.2 and Handlebars v4.0.10. To install handlebars:
```
Comment (calvinlfer) [inline README.md]: Why not merge this into module dependencies since both `handlebars` and `change-case` are module dependencies?

Comment (skantzidis) [reply README.md]: I have merged the dependencies into one heading "module dependencies". Fair comment.

File: README.md [hunk 2]
```diff
@@ -7,14 +7,24 @@ automate parts that require a bit more programmatic control.
...
 
 ```bash
 npm install -g handlebars
 ```
 
+The [change-case](https://www.npmjs.com/package/change-case) module is used to provide basic string manipulation functionality in our handlebars templates.
+Several custom handlebar functions are imported from this module. To install this module:
+```bash
+npm install -g change-case
```
Comment (hackmad) [inline README.md]: Does `change-case` need to be globally installed?

File: README.md (hunk 1 above)
Comment (calvinlfer) [reply README.md]: 👍

**Pull request closed without merging**